ConcernsCaseWork/ConcernsCaseWork.API.Contracts/Decisions/Outcomes/UpdateDecisionOutcomeRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConcernsCaseWork.API.Contracts.Decisions.Outcomes
+{
+	public record UpdateDecisionOutcomeRequest : DecisionOutcome
+	{
```

Comment:
Again, I'm guessing this is to be done later?

Reply:
Agreed offline